mergetools/kdiff3

```diff
@@ -25,3 +25,12 @@ merge_cmd () {
 exit_code_trustable () {
 	true
 }
+
+translate_merge_tool_path() {
+	if type -p kdiff3 >/dev/null 2>/dev/null
```

Comment:
Existing backend interfaces like bc and tortoisemerge uses a more portable "type" (without -p) for this purpose.  Is there a reason why this needs to use "-p"? If not, drop it.

Reply:
My fault, I suggested it to replace `which`...

Reply:
I have a mild prejudice against "whcih", which I found has been less reliable at least in the past (e.g. on Solaris I saw it exited with 0 status while giving "no such command" message, back when I still had access to several of them).

Reply:
changed to type (without -p).
Originally i had which because type has a totally different mening in a windows environment. If mergetool is called in a windows context, for example in backticks in a perl started from windows, the windows type command (= cat in linux/unix) will be called. However windows "type kdiff3" will also fail with nonzero exit (file not found), so everything is fine.

Reply:
`type` is a built-in of the Bash we're using to interpret the `mergetool` script, so there is no danger whatsoever that your command will run `type.exe` by mistake.